index.html

```diff
@@ -165,97 +165,136 @@ <h3 id="heading">Test Run for Firefox RHEL6.5 (Desktop)</h3>
-                    subtitle: 'shows detailed statuses per build and test run',
-                    img: 'images/screenshots/006_report_per_plan_and_build.png'
-                }],
-                imgIndex = 0,
-                imgsLength = imgs.length;
-
-            var heading = document.getElementById('heading'),
-                subtitle = document.getElementById('subtitle'),
-                nextBtn = document.getElementById('next'),
-                prevBtn = document.getElementById('prev'),
-                shown = document.getElementsByClassName('shown')[0],
-                hidden =  document.getElementsByClassName('hidden-right')[0],
-                index = document.getElementById('imgIndex');
-
-                index.textContent = (imgIndex + 1) + '/' + imgsLength;
-
-                shown.style.backgroundImage = 'url(' + imgs[0].img + ')';
-                heading.textContent = imgs[0].title;
-                subtitle.textContent = imgs[0].subtitle;
-
-                nextBtn.addEventListener('mouseover', function(){
-                    hidden.className = 'img hidden-right';
-                });
-
-                prevBtn.addEventListener('mouseover', function(){
-                    hidden.className = 'img hidden-left';
-                });
-
-                nextBtn.addEventListener('click', function(){
-                    imgIndex++;
-
-                    if(imgIndex > imgsLength - 1){
-                        imgIndex = 0;
-                    }
-
-                    index.textContent = (imgIndex + 1) + '/' + imgsLength;
-
-                    shown.className = 'img hidden-right';
-                    hidden.className = 'img shown';
-
-                    hidden.style.backgroundImage = 'url(' + imgs[imgIndex].img + ')';
-                    heading.textContent = imgs[imgIndex].title;
-                    subtitle.textContent = imgs[imgIndex].subtitle;
-
-                    shown = document.getElementsByClassName('shown')[0];
-                    hidden =  document.getElementsByClassName('hidden-right')[0];
-                });
-
-                prevBtn.addEventListener('click', function(){
-                    imgIndex--;
-
-                    if(imgIndex < 0){
-                        imgIndex = imgsLength - 1;
-                    }
-
-                    index.textContent = (imgIndex + 1) + '/' + imgsLength;
-
-                    shown.className = 'img hidden-left';
-                    hidden.className = 'img shown';
-
-                    hidden.style.backgroundImage = 'url(' + imgs[imgIndex].img + ')';
-                    heading.textContent = imgs[imgIndex].title;
-                    subtitle.textContent = imgs[imgIndex].subtitle;
-
-                    shown = document.getElementsByClassName('shown')[0];
-                    hidden =  document.getElementsByClassName('hidden-left')[0];
-                });
+        //Imgs array and other variables needed for the slider logic :)
+        var imgs = [
```

Comment:
@FreshyMc the JavaScript code should be indented 4 more spaces to the right, check out how the code was indented before your changes. When you do this then lines which have not been changed will not be shown as removed and the code review will be easier.